lib/oss/batch.rb

```diff
@@ -1,23 +1,38 @@
-require "#{Rails.root}/app/controllers/concerns/sqs_usable"
-include SqsUsable
+# require "#{Rails.root}/app/controllers/concerns/sqs_usable"
+# include SqsUsable
 
 class Batch
   def self.execute
     Oss::BatchLogger.info('Start convert process')
-    resp = receive_message(5)
-    if !resp || resp.messages.count == 0
+    resp = CloudConfig::SERVICE.receive_message(5)
+    unless CloudConfig::SERVICE.message_exist?(resp)
       Oss::BatchLogger.info('No SQS message found')
       return true
     end
-    resp.messages.each do |msg|
-      obj = JSON.parse(msg.body)
-      Oss::BatchLogger.info("Start converting slide. id=#{obj['id']} key=#{obj['key']}")
-      result = self.convert_slide(obj['key'])
-      if result
-        Oss::BatchLogger.info("Delete message from SQS. id=#{obj['id']} key=#{obj['key']}")
-        delete_message(msg)
-      else
-        Oss::BatchLogger.error("Slide conversion failed. id=#{obj['id']} key=#{obj['key']}")
+
+    if CloudConfig::service_name == 'aws'
```

Comment:
__[rubocop]__
* Style/ColonMethodCall: Do not use `::` for method calls. [:ref](https://github.com/bbatsov/ruby-style-guide#double-colons)